kitsune/sumo/static/sumo/less/main.less

```diff
@@ -1229,6 +1229,31 @@ input[type=button],
     position: relative;
     top: -2px;
   }
+
+  .firefox-download-button {
+    background: #81BC2E;
+    background: linear-gradient(to bottom, #81BC2E, #659324);
+    border-radius: 6px;
+    box-shadow: 0 3px 0 0 rgba(0,0,0,0.1), 0 -4px 0 0 rgba(0,0,0,0.1) inset;
+    display: inline-block;
+    line-height: 25px;
+    text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25);
+    transition: top 0s;
+    transition: all 0.5s;
+    width: 135px;
+    margin-top: 6px;
+    right: 6px;
+    position: relative;
+
+    .download-title {
+      color: #FFFFFF;
+      display: inline-block;
+      font-size: 77%;
+      font-family: georgia, "Bitstream Charter", serif;
```

Comment:
Using serif fonts is kind of jarring here, and feels a bit heavy to me. Maybe change this to look more like the download firefox button on https://support.mozilla.org/en-US/products/firefox ?